usr.sbin/bsdinstall/scripts/jail

```diff
@@ -108,20 +116,43 @@ fi
 
 bsdinstall checksum || error "Distribution checksum failed"
 bsdinstall distextract || error "Distribution extract failed"
-bsdinstall rootpass || error "Could not set root password"
+if [ -z "$SCRIPT" ]
```

Comment:
If this is to be conditional, it should use nonInteractive as well, so you can still script it to prompt if you wish